### DISCUSSION TITLE
Kafka consumer grupu

### DISCUSSION THREAD

**Discussion by amitmiglani91** [strimzi/strimzi-kafka-operator#7747] [General]
can you please provide me Strimzi cmd for deleting Consumer group.( kafka i have )

**Comment by scholzj** [comment]
There is no Strimzi command for it. Strimzi does not do anything with consumer groups.

**Reply by amitmiglani91** [reply]
Ok Thanx, means we have to do it in Pod itself, thanx for reply!!

**Reply by scholzj** [reply]
Keep i mind that ideally you should not run it directly from the broker pods to avoid running them out of memory etc. You can for example run it from a separate pod with `kubectl exec` ... something like:
```
kubectl run kafka-tools -ti --image=quay.io/strimzi/kafka:0.32.0-kafka-3.3.1 --rm=true --restart=Never -- bash
```
